Sources/Fuzzilli/Modules/Storage.swift

```diff
@@ -92,4 +81,17 @@ public class Storage: Module {
             logger.error("Failed to write program to disk: \(error)")
         }
     }
+
+    private func saveCorpus() {
```

Comment:
This function should now probably be called "saveState" :)

Reply:
Ah yep :)